### PR TITLE
Define swift include build components but dont wire them up

### PR DESCRIPTION
**Pull request by gottesmm**

### CODE REVIEW

**Comment by gottesmm** [comment]
I figured out the issue I ran into before. I was setting the initial value for include/build to be all components... but I was also asserting they were disjoint... *doh*.

@swift-ci Please smoke test and merge

File: cmake/modules/SwiftComponents.cmake [hunk 1]
```diff
@@ -67,10 +69,51 @@
...
     "A semicolon-separated list of components to install ${_SWIFT_DEFINED_COMPONENTS}")
+
+  # Make sure each component set does not have duplicates. This is necessary
+  # since these sets are represented in cmake as lists.
+  precondition_list_is_set(SWIFT_INCLUDE_COMPONENTS)
+  precondition_list_is_set(SWIFT_BUILD_COMPONENTS)
+  precondition_list_is_set(SWIFT_INSTALL_COMPONENTS)
+
+  # Then make sure that each one of our sets are disjoint.
+  precondition_list_is_disjoint(
```
Comment (dabrahams) [inline cmake/modules/SwiftComponents.cmake]: This assertioni started firing for me: https://ci.swift.org/job/swift-PR-Linux/3632/console

Did you make a mistake somewhere?

**Comment by gottesmm** [comment]
I am going to revert it. I am out and about.

**Comment by dabrahams** [comment]
Looks like we still have the problem: https://ci.swift.org/job/swift-PR-Linux/3632/console

**Pull request merged**